Resources/doc/5-automatic-route-generation_single-restful-controller.rst

```diff
@@ -31,6 +31,7 @@ are explained in the next section.
     # app/config/routing.yml
     users:
         type:     rest
+        host:     m.example.com
```

Comment:
I don't think we should include this in of our first examples, it would only confuse people as it's something very common.